src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -278,7 +278,7 @@ public static void DuplexCallback_Throws_FaultException_ReturnsFaultedTask()
     }
 
     [Fact]
-#if FEATURE_NETNATIVE
+#if FULLXUNIT_NOTSUPPORTED
     [ActiveIssue(833)] // Not supported in NET Native
```

Comment:
I know you have #1220 to track this... so does this test pass (can 833 be removed)? It's okay if you plan to remove 833 in another PR... just to confirm.